src/main/java/mate/academy/spring/service/UserMapper.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.UserResponseDto;
+
+public class UserMapper {
+
+    private UserMapper() {
+    }
+
+    public static UserResponseDto parse(User user) {
```

Comment:
It's bad practice to use static methods (only if it is absolutely necessary). Lets make this method not static